mesmer/core/auto_regression.py

```diff
@@ -2,6 +2,72 @@
 import xarray as xr
 
 
+def _draw_auto_regression_np(
+    *, intercept, coefs, covariance, n_samples, n_ts, seed, buffer
+):
+    """draw time series of an autoregressive process
```

Comment:
```suggestion
    """Draw time series of an autoregressive process plus innovations
```

Can we add a one line definition of 'innovations' in here (or a reference to somewhere which explains this term, I've never seen it before)